### ISSUE TITLE
File sharing server stops when the phone is idle

### ISSUE THREAD

**Issue by officiallor** [XITRIX/iTorrent#336]
Tried uploading something to the phone using the file sharing feature and after some time, when the phone was idle the web server stopped.

**Comment by XITRIX** [comment]
File sharing could be used only if app is in foreground, it does not allow app to work in background.
Otherwise iTorrent should constantly work in background which is malicious behaviour, because it will drain device's battery for nothing.

**Comment by officiallor** [comment]
It makes sense but it would be nice to have at least a notification that it will not work in background

**Comment by XITRIX** [comment]
Yes, but that mean that every time you close the app it will ring with notification, because app does not know if server is working or idle

Btw, I need to check if I can get this info from server, if yes, I could enable background for server if it is processing something until it will go idle
